p2p/src/net/libp2p/behaviour.rs

```diff
@@ -470,3 +421,33 @@ impl NetworkBehaviourEventProcess<mdns::MdnsEvent> for Libp2pBehaviour {
         }
     }
 }
+
+impl NetworkBehaviourEventProcess<connectivity::types::ConnectionManagerEvent> for Libp2pBehaviour {
+    fn inject_event(&mut self, event: connectivity::types::ConnectionManagerEvent) {
+        match event {
+            connectivity::types::ConnectionManagerEvent::Behaviour(event) => match event {
```

Comment:
What do you think about adding some usings in this function, so that match arms would be a little shorter? Something like that:
```rust
fn inject_event(&mut self, event: connectivity::types::ConnectionManagerEvent) {
    use connectivity::types::{ConnectionManagerEvent, BehaviourEvent};
    
    match event {
        ConnectionManagerEvent::Behaviour(event) => match event {
            BehaviourEvent::InboundAccepted { ... } => ...
        }
    }
}
```
Perhaps even wildcard import or something like `use connectivity::types::BehaviourEvent as BE` can be used here.

It is just a nitpick so feel free to ignore this comment. 🙃

Reply:
Yes good idea, the code is quite verbose. I'll fix them